runners/samza/src/main/java/org/apache/beam/runners/samza/translation/ParDoBoundMultiTranslator.java

```diff
@@ -274,10 +310,11 @@ public void translatePortable(
             windowedInputCoder.getValueCoder(), // input coder not in use
             windowedInputCoder,
             Collections.emptyMap(), // output coders not in use
-            Collections.emptyList(), // sideInputs not in use until side input support
+            new ArrayList<>(
+                sideInputMapping.values()), // sideInputs not in use until side input support
```

Comment:
Can use idToViewMapping.values() instead since the contents are the same

Reply:
Yup, both are fine I guess so I picked sideInputMapping map. We actually don't need to pass in both. There was some change from open source upstream to add the map along with the view list. We can refactor this  to remove this list in the future, but I prefer doing it in another patch since this one is already pretty big.